docs/debug.md

```diff
@@ -464,23 +448,23 @@ Recommendation:
 ```
 
 Ah, `xkcd-webserver` is a web server and we apparently forgot to include the
-`network-service` in our 'caps' in the yaml. At this point, you could simply
-add it to the `caps` for that service, rebuild the snap, remove the old snap
-and install the new one. Eg, after adding the `network-service` cap:
+`network-bind` in our 'plugs' in the yaml. At this point, you could simply
+add it to the `plugs` for that service, rebuild the snap, remove the old snap
+and install the new one. Eg, after adding the `network-bind` interface:
```

Comment:
e.g.; or e.g., 

http://english.stackexchange.com/questions/16172/should-i-always-use-a-comma-after-e-g-or-i-e